rul_datasets/reader/ncmapss.py

```diff
@@ -214,10 +217,10 @@ def prepare_data(self) -> None:
         """
         if not os.path.exists(self._NCMAPSS_ROOT):
             _download_ncmapss(self._NCMAPSS_ROOT)
-        if not os.path.exists(self._get_scaler_path()):
-            features, _, _ = self._load_data("dev")
-            scaler = scaling.fit_scaler(features, MinMaxScaler())
-            scaling.save_scaler(scaler, self._get_scaler_path())
+        #if not os.path.exists(self._get_scaler_path()):
```

Comment:
We will obviously need to find a solution for this. The scaling range should be part of the `scaler_path`. Try to modify `_get_scaler_path` to take the range into account. This way it would refit the scaler if the range changed.